docs/src/modules/components/AppWrapper.js

```diff
@@ -15,11 +14,6 @@ const jss = create({
   insertionPoint: process.browser ? document.querySelector('#insertion-point-jss') : null,
 });
 
-function themeSideEffect(reduxTheme) {
-  setPrismTheme(reduxTheme.paletteType === 'light' ? lightTheme : darkTheme);
-  document.body.dir = reduxTheme.direction;
```

Comment:
Yeah I should stick with the current approach and use useLayoutEffect. Still think that that the body.dir approach should be replaced. Especially since it's a theming thing and themes can vary in a page. We can follow up later.